README.md

```diff
@@ -1,5 +1,9 @@
 # EasyPost Java Client Library
 
+[![CI](https://github.com/EasyPost/easypost-java/workflows/CI/badge.svg)](https://github.com/EasyPost/easypost-java/actions?query=workflow%3ACI)
+[![CheckStyle](https://github.com/EasyPost/easypost-java/workflows/CheckStyle/badge.svg)](https://github.com/EasyPost/easypost-java/actions?query=workflow%3ACheckStyle)
```

Comment:
How does this badge even work if we only have a workflow titled `CI`? Is it a false positive green "passed"?

Reply:
My bad. It's not false, per se, we did have a standalone `CheckStyle` workflow when we first created it, before we folded it into the `CI` workflow, so it's outdated: https://github.com/EasyPost/easypost-java/actions/workflows/checkstyle.yml

Reply:
Ah, it's pulling the latest status from that orphaned workflow.